xml/ActivityStatus.xml

```diff
@@ -36,6 +38,7 @@
             </name>
             <description>
                 <narrative>Physical activity is complete or the final disbursement has been made, but the activity remains open pending financial sign off or M&amp;E</narrative>
+                <narrative xml:lang="fr">L'activité concrète est terminée ou le dernier débours a été effectué.</narrative>
```

Comment:
@Ocre42 because of changes to activity status code 3 and 4 the translations need to be changed:

3-  L'activité concrète est terminée ou le débours final a été effectué, mais l'activité reste ouverte en attendant une signature de l'autorité financière ou une évaluation (C et E)

4-L'activité concrète est terminée ou le dernier débours a été effectué.

Reply:
Done, switched them around